cmd/bosun/conf/conf.go

```diff
@@ -111,6 +111,16 @@ func (c *Conf) GraphiteContext() graphite.Context {
 	return graphite.Host(c.GraphiteHost)
 }
 
+// LogstashContext returns a Logstash service
+func (c *Conf) Logstash() expr.LogstashElasticHosts {
```

Comment:
Remove this function. Make c.LogstashElasticHosts of type expr.LogstashElasticHosts instead of []string.